docs/project/changelog.md

```diff
@@ -16,6 +16,9 @@ myst:
 
 ## Unreleased
 
+- {{ Enhancement }} Updated tqdm to version 4.66.5, fixing compatibility issues with Python 3.12
+  {pr}`5024`
+
```

Comment:
Could you please add an entry in the "Packages" heading too, below this?

Reply:
Done!